src/Sarif.UnitTests/Visitors/InsertOptionalDataVisitorTests.cs

```diff
@@ -162,38 +163,53 @@ private void ValidateResults(string output)
         }
 
         [Fact]
-        public void InsertOptionalDataVisitorTests_VisitDictionaryValueNullChecked_ValidEncoding()
+        public void InsertOptionalDataVisitorTests_ResolveOriginalUriBaseIds()
```

Comment:
>InsertOptionalDataVisitorTests_ResolveOriginalUriBaseIds [](start = 20, length = 56)

I advocate a `Noun_VerbPhrase` naming convention so the test name reads naturally in English. In this case, consider: `InsertOptionalDataVisitor_ResolvesOriginalUriBaseIds`.

Reply:
@michaelcfanning I wasn't clear. I was quibbling about the subject of the sentence being "the tests". It's not the tests that resolve the uriBaseIds, it's the visitor. Thus my naming suggestion `InsertOptionalDataVisitor_ResolvesOriginalUriBaseIds`.

---
In reply to: [214551684](https://github.com/Microsoft/sarif-sdk/pull/1062#discussion_r214551684) [](ancestors = 214551684,214520596)